doc/lightning-listconfigs.7.md

```diff
@@ -305,6 +305,7 @@ On success, an object is returned, containing:
 - **experimental-peer-storage** (boolean, optional): `experimental-peer-storage` field from config or cmdline, or default **deprecated, removal in v24.05** *(added v23.02)*
 - **experimental-quiesce** (boolean, optional): `experimental-quiesce` field from config or cmdline, or default **deprecated, removal in v24.05** *(added v23.08)*
 - **experimental-upgrade-protocol** (boolean, optional): `experimental-upgrade-protocol` field from config or cmdline, or default **deprecated, removal in v24.05** *(added v23.08)*
+- **unified-invoices** (boolean, optional): `unified-invoices` field from config or cmdline, or default *(added v23.11)*
```

Comment:
Should we call it `invoices-onchain-fallback`?  And I think it should merely be the default: we should be able to set it in a per-invoice basis as well.  Though that's a bit trickier; allowing `true` in fallbacks might break GRPC...

Reply:
Sure, sounds good.

Currently if you want to avoid using this fallback-tracking functionality you can pass in a fallback manually; this will result in the original behavior (and will also generate a warning).